### PR TITLE
Feature/event create form

### PR DESCRIPTION
**Pull request by linas19**

### CODE REVIEW

File: src/components/animal/events/AnimalEventForm.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import { makeStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+
+const useStyles = makeStyles({
+    margin: {
+        marginBottom: 24,
+        marginLeft: 15,
+        marginRight: 16,
```
Comment (benNek) [inline src/components/animal/events/AnimalEventForm.tsx:18]: you should use spacing variables provided by MaterialUI instead:
https://material-ui.com/system/spacing/

File: src/components/animal/events/AnimalEventForm.tsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    },
+    title: {
+        fontFamily: 'Roboto',
```
Comment (benNek) [inline src/components/animal/events/AnimalEventForm.tsx:21]: You should probably use according text variation instead and not apply those css props manually

File: src/components/animal/events/AnimalEventForm.tsx [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+    cancel: {
+        fontFamily: 'Roboto',
+        fontStyle: 'normal',
+        fontWeight: 500,
+        fontSize: 14,
+        lineHeight: 16,
+        textAlign: 'center',
+        letterSpacing: 1.25,
+        textTransform: 'uppercase',
+        color: '#13493B',
```
Comment (benNek) [inline src/components/animal/events/AnimalEventForm.tsx:38]: colors should be used from theme variable

File: src/components/animal/events/AnimalEventForm.tsx [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+        values.expense != null &&
+        values.comment != null &&
+        values.date != null;
+
+    return (
+        <div>
+            <Dialog open={open} onClose={handleClose}>
+                <DialogTitle className={classes.title}>Create new event</DialogTitle>
+                <FormControl variant="outlined" className={classes.margin}>
+                    <InputLabel htmlFor="outlined-age-native-simple">Type</InputLabel>
```
Comment (benNek) [inline src/components/animal/events/AnimalEventForm.tsx:83]: `outlined-age-native-simple` is not used in here.

File: src/components/animal/events/AnimalEventForm.tsx [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+                            id: 'typeOption-id',
+                        }}
+                    >
+                        {typeOptions.map(typeOption => (
+                            <MenuItem value={typeOption}>{typeOption}</MenuItem>
+                        ))}
+                    </Select>
+                </FormControl>
+                <FormControl variant="outlined" className={classes.margin}>
+                    <InputLabel htmlFor="outlined-age-native-simple">Category</InputLabel>
```
Comment (benNek) [inline src/components/animal/events/AnimalEventForm.tsx:99]: same for all others:)

File: src/components/animal/events/AnimalEventForm.tsx [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+                        variant="outlined"
+                        value={values.comment}
+                        onChange={handleChange}
+                        inputProps={{
+                            name: 'comment',
+                            id: 'comment-id',
+                        }}
+                    />
+                </FormControl>
+                <FormControl className={classes.margin}>
```
Comment (benNek) [inline src/components/animal/events/AnimalEventForm.tsx:142]: DatePicker could've been used instead

File: src/components/animal/events/AnimalEvents.tsx [hunk 7]
```diff
@@ -59,13 +62,59 @@ export default function AnimalEvents({ events }: AnimalEventsProps) {
...
+    const [open, setOpen] = React.useState(false);
+
+    const handleClickOpen = () => {
+        setOpen(true);
+    };
+
+    const handleClose = () => {
+        setOpen(false);
+    };
+    const typeOptions = [
```
Comment (benNek) [inline src/components/animal/events/AnimalEvents.tsx:74]: we usually move constants above the component, so they're not reinitialized on every rerender. also constants are tend to be UPPER_CASED

**Comment by saulyz** [comment]
Thanks for your efforts, we already have this functionality

**Pull request closed without merging**